wordpress-mobile/WordPress-iOS
# Blaze: Entry point analytics

Closes: #20090 
Closes: #20091
Closes: #20092 

- Adds event tracking for Blaze entry points

## How to test

0. Make sure that Blaze is enabled via the debug menu
1. Switch to a site that has Blaze enabled - let's call this site A (See peeHDf-zm-p2 for more info on Blaze requirements)
2. Switch to the dashboard
3. ✅ `blaze_feature_displayed <source: dashboard_card>` is tracked
5. Tap on the Blaze dashboard card
6. ✅ `blaze_feature_tapped <source: dashboard_card>` is tracked
7. Tap on the ellipsis buton on the Blaze dashboard card
8. ✅ `blaze_feature_menu_accessed <source: dashboard_card>` is tracked
9. Tap on "Hide this"
10. ✅ `blaze_feature_hide_tapped <source: dashboard_card>` is tracked
12. Switch to the site menu
13. ✅ `blaze_feature_displayed <source: menu_item>` is tracked
14. Tap on the Blaze menu item
15. ✅ `blaze_feature_tapped <source: menu_item>` is tracked
16. Tap on the Posts menu item
17. On a Blaze enabled post, tap on the ellipsis button
18. ✅ `blaze_feature_displayed <source: posts_list>` is tracked
19. Tap on "Promote with Blaze"
20. ✅ `blaze_feature_tapped <source: posts_list>` is tracked


## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
